src/bot/cogs/sgc.py

```
@@ -0,0 +1,419 @@
+
+            ref_author = None
+            ref_content = None
+            if ref_mid and ref_mid != "0":
+                async for past_message in message.channel.history(limit=100):
+                    try:
+                        p_dic = json.loads(past_message.content)
+                        if p_dic.get("messageId") == str(ref_mid):
+                            ref_author = f"{p_dic.get('userName')}#{p_dic.get('userDiscriminator')}"
+                            ref_content = p_dic.get("content", "内容なし")
+                            break
+                    except: continue
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)